README.md

```diff
@@ -25,11 +25,29 @@ Plugin package to bridge the native Appcues SDKs in a Flutter application.
 
 ## 🚀 Getting Started
 
+### Prerequisites
+
+Android: your application's `build.gradle` must have a `compileSdkVersion` of 32+ and `minSdkVersion` of 21+
+```
+android {
+    compileSdkVersion 32    
+
+    defaultConfig {
+        minSdkVersion 21
+    }
+}
+```
+
+iOS: your application must target iOS 11+ to install the SDK, and iOS 13+ to render Appcues content.  In the application's `Podfile`, include at least this minimum version.
+```
+platform :ios, '11.0'
```

Comment:
The customer might need to update their xcodeproj too?

Reply:
yes good call! updated to add that.